modules/ngscheckmate/ncm/main.nf

```diff
@@ -0,0 +1,50 @@
+process NGSCHECKMATE_NCM {
+    label 'process_low'
+
+    conda (params.enable_conda ? "bioconda::ngscheckmate=1.0.0" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'quay.io/biocontainers/ngscheckmate:1.0.0--py27r41hdfd78af_3':
+        'quay.io/biocontainers/ngscheckmate:1.0.0--py27r41hdfd78af_3' }"
+
+    input:
+    path files
+    path snp_bed
+    path fasta
+
+    output:
+    path "*.pdf"            , emit: pdf
+    path "*_corr_matrix.txt", emit: corr_matrix
+    path "*_matched.txt"    , emit: matched
+    path "*_all.txt"        , emit: all
+    path "versions.yml"     , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "output"
+    def unzip = files.any { it.toString().matches(".*\\.vcf\\.gz") }
```

Comment:
```suggestion
    def unzip = files.any { it.toString().endsWith(".vcf.gz") }
```
Sorry, if it's only at least one file ends in vcf.gz, then it's nicer to use endsWith as it's simpler to read. If a regex needs to be matched, then use `matches`.